testing/tests/DevExpress.ui.widgets/listParts/commonTests.js

```diff
@@ -1788,6 +1789,150 @@ QUnit.module('selection', moduleSetup, () => {
     });
 });
 
+QUnit.module('selectByClick option', {
+    beforeEach: function() {
+        this.onSelectionChangedSpy = sinon.spy();
+        this.items = [1, 2];
+        this.createList = (selectionMode, selectByClick) => {
+            return $('#list').dxList({
+                items: this.items,
+                showSelectionControls: true,
+                selectByClick,
+                selectionMode,
+                onSelectionChanged: this.onSelectionChangedSpy,
+            });
+        };
+    } }, ()=> {
+
+    QUnit.test('onSelectionChanged event should call on item click if selectionMode=single, selectByClick=true', function(assert) {
```

Comment:
If u name tests like "onSelectionChanged should call" u should test only that selectionChanged is called.
If u wanna test class toggling and event firing in a single test, let's name it like
```suggestion
    QUnit.test('selection should be changed on item click if selectionMode=single, selectByClick=true', function(assert) {
```

Please, update all your tests

Reply:
Fixed